docs/standard/io/snippets/pipes/pipes-from-streams/Program.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Buffers;
+using System.IO;
+using System.IO.Pipelines;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+class Program
+{
+    static async Task Main()
+    {
+        using var stream = new StreamReader("lorem-ipsum.txt");
+        using var cts = new CancellationTokenSource();
```

Comment:
What is this used for?

Reply:
I wouldn't use both this and  CancelPendingRead and CancelPendingFlush, you don't need all 3, you should either use the token or use the other methods.

Reply:
The console application supports cancellation when a user presses <kbd>C</kbd>. The corresponding `cts.Token` flows through the various calls and signals cancellation. Should we do something different here?

Reply:
This is good to know, I didn't realize that! So, you're supposed to do one or the other, but not both.